service/awsconfig/v10/ebs/error.go

```diff
@@ -22,6 +30,9 @@ func IsVolumeNotFound(err error) bool {
 	if aerr.Code() == "NotFound" {
 		return true
 	}
+	if c == volumeNotFoundError {
+		return true
+	}
```

Comment:
I think this must go before the type assertion because `volumeNotFoundError` is not an AWS error type and thus in case of `volumeNotFoundError` we will always get false now.

Reply:
Thanks good catch and I've moved the type assertion up.